Firebase/ios/BonfireTests/TestScheduler+Extensions.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import RxSwift
+import RxTests
+
+extension TestScheduler {
+    func singleEventAndComplete<Element>(element: Element) -> Observable<Element> {
+        return self.createColdObservable([
+            next(0, element),
+            completed(0)
+            ]).asObservable()
+    }
+
+    func singleEventAndHang<Element>(element: Element) -> Observable<Element> {
```

Comment:
Do we need to rename this? Or is it descriptive enough? Basically want to simulate a hot observable that stays open because more data might come out later (like when we observe a node in Firebase).